crypto/crypto_test.ts

```diff
@@ -260,6 +266,35 @@ Deno.test("digest() keeps memory usage reasonable with many calls", async () =>
   );
 });
 
+Deno.test("digest() throws on invalid input", async () => {
+  const inputString = "taking the hobbits to isengard";
+  const inputBytes = new TextEncoder().encode(inputString);
+
+  await assertRejects(
+    () => stdCrypto.subtle.digest("BLAKE2B", {} as Iterable<never>),
```

Comment:
Nit: just to make it clearer to the reader that this is an async method.
```suggestion
    async () => await stdCrypto.subtle.digest("BLAKE2B", {} as Iterable<never>),
```